lib/config/presets/local.ts

```diff
@@ -0,0 +1,20 @@
+import { Preset } from './common';
+import * as gitlab from './gitlab';
+import * as github from './github';
+import { RenovateConfig } from '../common';
+
+export async function getPreset(
+  pkgName: string,
+  presetName = 'default',
+  baseConfig: RenovateConfig
+): Promise<Preset> {
+  if (baseConfig.platform?.toLowerCase() === 'gitlab') {
+    return gitlab.getPreset(pkgName, presetName, baseConfig.endpoint);
```

Comment:
I am unsure whether it is good to have the switch here vs. moving this logic (or parts of it) to the `platform` module. What do you think?

Reply:
Let's keep a switch here for now. We support 6 platforms now but only 2 of them with presets, so I think it's ok to keep it in the presets logic.

Reply:
Alright!